build-tools/packages/build-cli/src/commands/generate/typetests.ts

```diff
@@ -75,7 +83,8 @@ export default class GenerateTypetestsCommand extends PackageCommand<
 		// Do not check that file exists before opening:
 		// Doing so is a time of use vs time of check issue so opening the file could fail anyway.
 		// Do not catch error from opening file since the default behavior is fine (exits process with error showing useful message)
-		const currentPackageJson = pkg.packageJson;
+		const currentPackageJson = pkg.packageJson as PackageWithTypeTestSettings;
```

Comment:
Same as the previous comment, assuming it works.
```suggestion
		const currentPackageJson: PackageWithTypeTestSettings = pkg.packageJson;
```